src/main/java/net/notfab/lindsey/core/commands/wiki/Hearthstone.java

```diff
@@ -49,59 +49,59 @@ public CommandDescriptor getInfo() {
+        String cardname = "";
+        String locale = profile.getLanguage().name().replaceAll("_", "");
+        boolean gold = false;
+
+        if (args.length == 1) {  //Available locales: enUS, enGB, deDE, esES, esMX, frFR, itIT, koKR, plPL, ptBR, ruRU, zhCN, zhTW, jaJP, thTH.
+            cardname = args[0];
+        } else if (args.length == 2) {
+            cardname = args[0];
+            if (args[1].equals("true")) {
+                gold = true;
             }
-
-            Request request = new Request.Builder()
-                    .url("https://omgvamp-hearthstone-v1.p.rapidapi.com/cards/search/" + cardname + "?collectible=1&locale=" + locale)
-                    .get()
-                    .addHeader("x-rapidapi-host", "omgvamp-hearthstone-v1.p.rapidapi.com")
-                    .addHeader("x-rapidapi-key", key)
-                    .build();
-            Response resp = client.newCall(request).execute();
-            String str = resp.body().string();
-
-            String result = "";
-            try {
-                JSONArray arr = new JSONArray(str);
-                if (gold) {
-                    result = arr.getJSONObject(0).getString("imgGold");
-                } else {
-                    result = arr.getJSONObject(0).getString("img");
-                }
-            } catch (JSONException e) {
-                JSONObject obj = new JSONObject(str);
-                if (obj.getInt("error") == 404) {
-                    result = i18n.get(member, "commands.wiki.hearthstone.404");
-                }
+        } else if (args.length == 3) {
+            cardname = args[0];
+            locale = args[2];
+            if (args[1].equals("true")) {
+                gold = true;
             }
+        }
 
-            msg.send(channel, result);
-        } else {
-            HelpArticle article = this.help(member);
-            article.send(channel, member, args, msg, i18n);
+        Request request = new Request.Builder()
+                .url("https://omgvamp-hearthstone-v1.p.rapidapi.com/cards/search/" + cardname + "?collectible=1&locale=" + locale)
+                .get()
+                .addHeader("x-rapidapi-host", "omgvamp-hearthstone-v1.p.rapidapi.com")
+                .addHeader("x-rapidapi-key", key)
+                .build();
+        Response resp = client.newCall(request).execute();
+        String str = resp.body().string();
+
+        String result = "";
+        try {
+            JSONArray arr = new JSONArray(str);
+            if (gold) {
+                result = arr.getJSONObject(0).getString("imgGold");
+            } else {
+                result = arr.getJSONObject(0).getString("img");
+            }
+        } catch (JSONException e) {
+            JSONObject obj = new JSONObject(str);
+            if (obj.getInt("error") == 404) {
+                result = i18n.get(member, "commands.wiki.hearthstone.404");
+            }
         }
 
+        msg.send(channel, result);
```

Comment:
If a JSONException is thrown, that does not have the error code 404, an empty message will be sent.